torchrl/envs/transforms/transforms.py

```diff
@@ -1279,18 +1279,62 @@ def __init__(
                 "next_observation_state",
             ]
         super().__init__(in_keys=in_keys)
-        if not isinstance(loc, torch.Tensor):
+        self.standard_normal = standard_normal
+        self.eps = 1e-6
+
+        if loc is not None and not isinstance(loc, torch.Tensor):
             loc = torch.tensor(loc, dtype=torch.float)
-        if not isinstance(scale, torch.Tensor):
+
+        if scale is not None and not isinstance(scale, torch.Tensor):
             scale = torch.tensor(scale, dtype=torch.float)
+            scale.clamp_min(self.eps)
 
         # self.observation_spec_key = observation_spec_key
-        self.standard_normal = standard_normal
         self.register_buffer("loc", loc)
-        eps = 1e-6
-        self.register_buffer("scale", scale.clamp_min(eps))
+        self.register_buffer("scale", scale)
+
+    def init_stats(
+        self,
+        num_iter: int,
+        reduce_dim: Union[int, Tuple[int]] = 0,
+        key: Optional[str] = None,
+    ) -> None:
+        if self.loc is not None or self.scale is not None:
+            raise RuntimeError(
```

Comment:
we could test that we reach this error message by adding one more `init_stats` in the tests just after we call it:
```python
t_env.transform.init_stats(num_iter=11)
with pytest.raises(RuntimeError, match="already initialized"):
    t_env.transform.init_stats(num_iter=11)
```